tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "emitDecoratorMetadata": true,
     "noUnusedLocals": true,
     "skipLibCheck": true,
+    "esModuleInterop": true,
```

Comment:
This is needed for the new imported lib?

Reply:
yep, otherwise it throws that .default is undefined error